content/millennium/dstu2/individuals/patient.md

```diff
@@ -145,6 +149,88 @@ The ability to perform patient combine or uncombine operations is not available
 
 The common [errors] may be returned.
 
+## Create
+
+Create an individual Patient:
+
+    POST /Patient
+
+_Implementation Notes_
+
+* The following elements are not supported and will cause an error to be returned if set:
+    * [Patient.deceased]
+    * [Patient.multipleBirth]
+    * [Patient.photo]
+    * [Patient.contact]
+    * [Patient.animal]
+    * [Patient.managingOrganization]
+    * [Patient.link]
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', Accept: 'application/json+fhir', 'Content-Type': 'application/json+fhir'} %>
+
+### Body fields
+
+Notes:
+
+* Birth Sex may be recorded as an extension.
```

Comment:
Could this be a note in the table? Or is it here to draw more attention?

Reply:
I did intend to call it out here because it isn't listed as a field on the FHIR spec. I can remove it if it looks awkward here.